.github/workflows/ci.yml

```diff
@@ -1,92 +1,84 @@
     services:
       postgres:
-        image: postgres
+        image: postgres:9.6
         env:
           POSTGRES_USER: 'postgres'
           POSTGRES_HOST_AUTH_METHOD: 'trust'
-        options: >-
-          --health-cmd pg_isready
-          --health-interval 10s
-          --health-timeout 5s
-          --health-retries 3
+        # Health check to wait for postgres to start.
         ports:
           - 5432:5432
-
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 3
       mariadb:
-        image: mariadb
+        image: mariadb:10.5
         env:
           MYSQL_USER: 'root'
           MYSQL_ALLOW_EMPTY_PASSWORD: "true"
         ports:
           - 3306:3306
-        options: >-
-          --health-cmd="mysqladmin ping"
-          --health-interval 10s
-          --health-timeout 5s
-          --health-retries 3
-
+        options: --health-cmd="mysqladmin ping" --health-interval 10s --health-timeout 5s --health-retries 3
     strategy:
       fail-fast: false
       matrix:
-        database: ['pgsql', 'mariadb']
-        moodle-branch: ['MOODLE_39_STABLE', 'MOODLE_310_STABLE', 'master']
-        node: ['14.15.1']
-        php: ['7.3', '7.4']
         include:
-          - {moodle-branch: 'MOODLE_37_STABLE', php: '7.1', node: '14.15.1', database: 'mariadb'}
-          - {moodle-branch: 'MOODLE_37_STABLE', php: '7.1', node: '14.15.1', database: 'pgsql'}
-
+          - {moodle-branch: 'MOODLE_33_STABLE', php: '7.1', node: '8.9', database: 'pgsql'}
+          - {moodle-branch: 'MOODLE_34_STABLE', php: '7.2', node: '8.9', database: 'pgsql'}
+          - {moodle-branch: 'MOODLE_35_STABLE', php: '7.2', node: '14.15', database: 'pgsql'}
+          - {moodle-branch: 'MOODLE_36_STABLE', php: '7.3', node: '8.9', database: 'pgsql'}
+          - {moodle-branch: 'MOODLE_37_STABLE', php: '7.1', node: '14.15', database: 'mariadb'}
+          - {moodle-branch: 'MOODLE_37_STABLE', php: '7.3', node: '14.15', database: 'pgsql' }
+          - {moodle-branch: 'MOODLE_38_STABLE', php: '7.1', node: '14.15', database: 'mariadb' }
+          - {moodle-branch: 'MOODLE_38_STABLE', php: '7.3', node: '14.15', database: 'pgsql' }
+          - {moodle-branch: 'MOODLE_39_STABLE', php: '7.2', node: '14.15', database: 'mariadb'}
+          - {moodle-branch: 'MOODLE_39_STABLE', php: '7.3', node: '14.15', database: 'pgsql'}
+          - {moodle-branch: 'MOODLE_39_STABLE', php: '7.4', node: '14.15', database: 'mariadb'}
+          - {moodle-branch: 'MOODLE_39_STABLE', php: '7.4', node: '14.15', database: 'pgsql'}
+          - {moodle-branch: 'MOODLE_310_STABLE', php: '7.4', node: '14.15.1', database: 'mariadb'}
+          - {moodle-branch: 'MOODLE_310_STABLE', php: '7.4', node: '14.15.1', database: 'pgsql'}
+          - {moodle-branch: 'MOODLE_311_STABLE', php: '7.4', node: '14.15.1', database: 'mariadb'}
+          - {moodle-branch: 'MOODLE_311_STABLE', php: '7.4', node: '14.15.1', database: 'pgsql'}
```

Comment:
might be better to change this back to the matrix that was there before.....this list will go out of control with new versions getting added in the future.....For a working example please refer to these changes, which Anu added recently: https://github.com/catalyst/moodle-auth_enrolkey/blob/master/.github/workflows/38-master.yml#L38-L49

Reply:
Thanks for the link. I've fixed this here: https://github.com/catalyst/moodle-local_aws/blob/update_to_3.208.1/.github/workflows/ci.yml#L27-L40